net/Socket.cpp

```diff
@@ -569,7 +650,7 @@ int SocketPoll::poll(int64_t timeoutMaxMicroS)
                     rc = -1;
                 }
 
-                if (!disposition.isContinue())
+                if (!_pollSockets[i]->isOpen() || !disposition.isContinue())
```

Comment:
Given the above comparison - we expect handlePoll to close the socket, but not set the disposition correctly ? what is the use-case for that; should we just assert that if it isContinue then it must be isOpen instead ?

Reply:
`!_pollSockets[i]->isOpen()` is redundant indeed. Iff always `disposition.isContinue() == _pollSockets[i]->isOpen()` (as it probably should be aligned) then the whole block is redundant.
However, I am not sure whether it always gets closed with the disposition in all cases. So I would prefer to keep it in or analyze better..

Reply:
great =)